buddyboss/buddyboss-platform
# PROD-725 Group type shortcode empty space at titlebar - fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3033  .

### How to test the changes in this Pull Request:

1. Disable buddypanel
2. Add a group type shortcode in a page
3. View the page
4. See issue not happening

### Proof Screenshots or Video

![screenshot-localhost-2021 10 08-01_25_03](https://user-images.githubusercontent.com/12556175/136449688-e2d6eb05-ddc6-4190-a14d-4fa20eec2880.png)


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
